numpy/core/tests/test_multiarray.py

```diff
@@ -3708,7 +3708,7 @@ def test_highest_available_pickle_protocol(self):
         except ImportError:
             pickle5 = None
 
-        if sys.version_info[:2] >= (3, 8) or pickle5 is not None:
+        if sys.version_info[:4] >= (3, 8, 0, "beta") or pickle5 is not None:
```

Comment:
Sorry, assuming beta release will have it for now of course. But we can wait for it to actually happen (and I can remove this again for 1.17 if it is unclear).

Reply:
I don't worry too much about Python pre-releases until they get near the end, too much changes in the interim.

Reply:
Does that mean you prefer not to include the "beta" stuff? I suppose I could just rewrite the test to be python version agnostic, it doesn't really matter much that we check if python implements the feature...

Reply:
I agree with the last sentiment - our tests should ideally not include tests of what some python version includes.

Reply:
Just removed the whole test. It isn't really our business to test that the experimental pickle5 module manages to successfully inject into pickle, or was there a good reason for this @pitrou?

Reply:
@seberg That sounds right to me.